packages/mobile/src/harmony-native/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -54,9 +54,9 @@ export const SelectablePill = (props: SelectablePillProps) => {
   const tap = Gesture.Tap()
     .shouldCancelWhenOutside(true)
     .onBegin(() => {
-      pressed.value = withTiming(1, motion.hover)
+      pressed.value = withTiming(1, motion.press)
```

Comment:
🙈

Reply:
(technically the same value, but yes 🙈 )